### PR TITLE
Updating Jira Fix Version details

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/process/overview.rst [hunk 1]
```diff
@@ -25,23 +25,37 @@ Mattermost priorities are managed in Jira tickets, which are created by the core
...
+Developers, Product Managers, Release and Quality Assurance teams use the Fix Version field for the following pruposes:
+
+ Developers: 
+  - Fix Version field is the one that shows up on individual tickets in the backlog view.
+ Product Managers:
+  - 
+ Release Manager:
+  - Follow status of resolved, open and submitted tickets for current release.
+  - Compose various release metrics.
+ Quality Assurance:
```
Comment (amyblais) [inline source/process/overview.rst]: @lindalumitchell Please fill in any other use cases where QAs use the Jira Fix Version.

File: source/process/overview.rst [hunk 2]
```diff
@@ -25,23 +25,37 @@ Mattermost priorities are managed in Jira tickets, which are created by the core
...
+Release planning sets the "Fix Version" of tickets to one of the upcoming monthly releases. The Fix Version is an estimate of when a feature might ship, which may change as the planning process continues, until the ticket is scheduled for a Sprint. 
+
+Jira Fix Version
+~~~~~~~~~~~~~~~~
+
+Developers, Product Managers, Release and Quality Assurance teams use the Fix Version field for the following pruposes:
+
+ Developers: 
+  - Fix Version field is the one that shows up on individual tickets in the backlog view.
+ Product Managers:
```
Comment (amyblais) [inline source/process/overview.rst]: @thefactremains @wiersgallak @esethna @jasonblais @aaronrothschild Please fill in any use cases where PMs use the Jira Fix Version.

File: source/process/overview.rst [hunk 3]
```diff
@@ -25,23 +25,37 @@ Mattermost priorities are managed in Jira tickets, which are created by the core
...
-Release planning sets the "Fix Version" of tickets to one of the upcoming monthly releases. The Fix Version is an estimate of when a
-feature might ship, which may change as the planning process continues, until the ticket is scheduled for a Sprint.
+Release planning sets the "Fix Version" of tickets to one of the upcoming monthly releases. The Fix Version is an estimate of when a feature might ship, which may change as the planning process continues, until the ticket is scheduled for a Sprint. 
+
+Jira Fix Version
+~~~~~~~~~~~~~~~~
+
+Developers, Product Managers, Release and Quality Assurance teams use the Fix Version field for the following pruposes:
+
+ Developers:
```
Comment (amyblais) [inline source/process/overview.rst]: @grundleborg @jwilander @lieut-data @enahum @levb Please fill in any other use cases where Devs use the Jira Fix Version.

File: source/process/overview.rst (hunk 2 above)
Comment (jasonblais) [reply source/process/overview.rst]: Some ways I've used in the past
1 - Use fix version to prioritize backlog
2 - Use fix version to estimate/plan what stories ship in what release

^I don't think either would need to be tied to specific version number; e.g. the first could be accomplished with priority field or near-term/long-term/future fix versions.

File: source/process/overview.rst (hunk 3 above)
Comment (jwilander) [reply source/process/overview.rst]: For stories, we use them for soft guidelines for what is targeted for what release. For bugs, we use them to know which bugs are priority based on where we are in the release cycle.

File: source/process/overview.rst (hunk 1 above)
Comment (lindalumitchell) [reply source/process/overview.rst]: Thanks @amyblais, I would add and slightly adjust QA items like this:
- Prioritize testing of tickets and associated PRs for the next release
- Assess completeness of a release before cutting final
- Track when changes actually went in, to help investigate bugs and expected behavior
- Report on issues, changes, and tests by release

File: source/process/overview.rst [hunk 4]
```diff
@@ -25,23 +25,37 @@ Mattermost priorities are managed in Jira tickets, which are created by the core
...
 ~~~~~~~~~~~~~~~~
 
-Release planning sets the "Fix Version" of tickets to one of the upcoming monthly releases. The Fix Version is an estimate of when a
-feature might ship, which may change as the planning process continues, until the ticket is scheduled for a Sprint.
+Release planning sets the "Fix Version" of tickets to one of the upcoming monthly releases. The Fix Version is an estimate of when a feature might ship, which may change as the planning process continues, until the ticket is scheduled for a Sprint. 
+
+Jira Fix Version
+~~~~~~~~~~~~~~~~
+
+Developers, Product Managers, Release and Quality Assurance teams use the Fix Version field for the following pruposes:
```
Comment (lindalumitchell) [inline source/process/overview.rst]: Small typo: should be "purposes".

**Pull request merged**